### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by InspectorGadget**

### CODE REVIEW

File: src/Niekert/Discord/Main.php [hunk 1]
```diff
@@ -18,12 +19,31 @@ public function onEnable(){
...
+                }
+                else {
+                
+                    $this->reloadConfig();
+                    $data = array("content" => $message, "username" => "$username");
+                    $curl = curl_init("$webhook");
+                    curl_setopt($curl, CURLOPT_CUSTOMREQUEST, "POST");
+                    curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
+                    curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+                    return curl_exec($curl);
```
Comment (Thunder33345) [inline src/Niekert/Discord/Main.php:42]: why even return curl_exec?
onEnable should be void not mixed
You are also suppose to use a Async task so server wont stop loading

File: src/Niekert/Discord/Main.php [hunk 2]
```diff
@@ -18,12 +19,31 @@ public function onEnable(){
...
+                    
+                    // Let you know here!
+                    $this->getLogger()->warning('Check your Discord Server now :)');
```
Comment (Thunder33345) [inline src/Niekert/Discord/Main.php:45]: Dead Code due to the return

File: src/Niekert/Discord/Main.php [hunk 3]
```diff
@@ -18,12 +19,31 @@ public function onEnable(){
...
+                }
+                elseif ($webhook === "") {
+                    $this->getLogger()->warning('Please set your webhook in config.yml');
+                }
+                elseif ($message === "") {
+                    $this->getLogger()->warning('Please set your message in config.yml');
+                }
+                else {
+                
+                    $this->reloadConfig();
```
Comment (Thunder33345) [inline src/Niekert/Discord/Main.php:36]: why reload config?

File: src/Niekert/Discord/Main.php [hunk 4]
```diff
@@ -18,12 +19,31 @@ public function onEnable(){
...
+                    $data = array("content" => $message, "username" => "$username");
+                    $curl = curl_init("$webhook");
```
Comment (Thunder33345) [inline src/Niekert/Discord/Main.php:38]: no need for "$webhook" why not just $webhook

File: src/Niekert/Discord/Main.php [hunk 5]
```diff
@@ -18,12 +19,31 @@ public function onEnable(){
...
+                elseif ($webhook === "") {
+                    $this->getLogger()->warning('Please set your webhook in config.yml');
+                }
+                elseif ($message === "") {
+                    $this->getLogger()->warning('Please set your message in config.yml');
+                }
+                else {
+                
+                    $this->reloadConfig();
+                    $data = array("content" => $message, "username" => "$username");
```
Comment (Thunder33345) [inline src/Niekert/Discord/Main.php:37]: If you are reloading config you are suppose to re fetch the variables or there's no point in reloading it

**Review by Thunder33345 — CHANGES_REQUESTED** [review]
Need some fixing

**Comment by InspectorGadget** [comment]
Please refer to https://github.com/niekmussche/Discord-MCPE/pull/4 as this branch got messed up

**Pull request closed without merging**